src/handlers/query-wallet.ts

```diff
@@ -30,15 +31,21 @@ export async function registerWallet(context: Context, body: string) {
   }
 
   if (!address) {
-    return context.logger.info("Skipping to register a wallet address because both address/ens doesn't exist");
+    const message = "Skipping to register a wallet address because both address/ens doesn't exist";
+    await addCommentToIssue(context, `\`\`\`diff\n# ${message}`);
```

Comment:
If you use the logger you should not have to manually build these string, use the returned value of the logger instead.